.github/scripts/classify-wework-desktop-e2e.sh

```
@@ -308,6 +309,19 @@ classify_wework_path() {
       return
       ;;
 
+    # Plugin development verifies the installed Codex guide, generated local
+    # project, conditional conversation sidebar, isolated Wework, and HMR.
+    wework/dsh/plugin-developer/* | \
+      wework/e2e/desktop/scenarios/plugin-development.scenario.mjs | \
+      wework/electron/src/runtime/plugin-development-manager* | \
+      wework/resources/bundled-plugins/wework-personal/plugins/wework-plugin-developer/* | \
+      wework/src/features/dsh-plugins/pluginDevelopment* | \
+      wework/src/components/layout/workspace-panels/rightWorkspaceDshSidebar* | \
+      wework/src/components/layout/workspace-panels/RightWorkspacePanel.tsx)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 537.

(SC2221)